pkg/config/config.go

```diff
@@ -373,13 +376,13 @@ func (e *Edge) FormatValues(record base.Record) string {
 		//TODO(yee): differentiate string and integer column type, find and compare src/dst vertex column with property
 		srcVID = fmt.Sprintf("%s(%q)", *e.SrcVID.Function, record[*e.SrcVID.Index])
 	} else {
-		srcVID = record[*e.SrcVID.Index]
+		srcVID = base.TryConvInt64(record[*e.SrcVID.Index])
```

Comment:
Nebula will support string type for VID in next version. So, currently, I suggest you use function to convert string to integer, even if the string content of vid is all digitals.